tests/CMakeLists.txt

```diff
@@ -40,6 +40,7 @@ if(${CHFL_TEST_RUNNER} STREQUAL "valgrind")
         RUNNER_COMMAND
         "valgrind" "--leak-check=full" "--dsymutil=yes" "--error-exitcode=125"
         "--suppressions=${CMAKE_CURRENT_SOURCE_DIR}/valgrind.supp"
+        "--track-origins=yes" "--leak-check=full" "--show-leak-kinds=all"
```

Comment:
Should `--leak-check=full` be there twice?

Reply:
whoops, that was debug code that should not have been included ...